spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/tracing/OpenTelemetryAutoConfiguration.java

```diff
@@ -34,6 +35,7 @@
  */
 @AutoConfiguration(before = MicrometerTracingAutoConfiguration.class)
 @Import({ SdkConfiguration.class, TracerConfiguration.class, MicrometerConfiguration.class })
+@EnableConfigurationProperties(TracingProperties.class)
```

Comment:
Why is this needed? It's duplicating the same from `TracerConfiguration`.

Reply:
`TracerConfiguration` doesn't have this. After this change it looks exactly the same way as `BraveAutoConfiguration` looks like